pkg/cloudprovider/controller/nodecontroller.go

```diff
@@ -80,6 +80,61 @@ func NewNodeController(
 	}
 }
 
+// Generates num pod CIDRs that could be assigned to nodes.
+func generateCIDRs(num int) util.StringSet {
+	res := util.NewStringSet()
+	for i := 0; i < num; i++ {
+		// TODO: Make the CIDRs configurable.
+		res.Insert(fmt.Sprintf("10.244.%v.0/24", i))
```

Comment:
I'm saying that I might get around to checking that in before this PR even lands, so this could end up being a regression in configurability. It's only #4596 that stopped me from checking the rest of that script in (that commit is sitting in the same branch, for context).